app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsViewModel.kt

```diff
@@ -62,7 +62,7 @@ class PullRequestsViewModel @Inject constructor(
         fetchData()
     }
 
-    private fun fetchData() {
+    fun fetchData() {
```

Comment:
question: if you re-used the same method for fetching isn't a problem, where data disappears when user does pull-to-refresh? Usually when user doesn't pull-to-refresh, the old data should be still presented.

suggestion: If you make this class open, it would be beneficial adding unit tests for this class.

Reply:
No, it's not. When user doesn't pull to refresh, the old data is still presented.

Reply:
I don't think this test is necessary because the `WHEN init THEN display pull requests list` test is doing this job.

Reply:
> No, it's not. When user doesn't pull to refresh, the old data is still presented.
I'm talking about this full-screen loader, when doing pull-to-refresh:
[full-screen-loader.webm](https://github.com/appunite/Loudius/assets/689211/87f94da5-c231-419c-97e1-782e112ab02e)

Reply:
Ok, now I understand :-)